logik/tools/fasm_to_bitstream/bitstream_finish.py

```diff
@@ -24,6 +24,10 @@ def setup(chip):
              ",".join(['fpga', part_name, 'file', 'bitstream_map']),
              step=step, index=index)
 
+    chip.add('tool', tool, 'task', task, 'input', f'{chip.top()}.fasm', step=step, index=index)
```

Comment:
Just to confirm, these additions are to help enforce that these files are present/generated?

Reply:
these are needed to do error checking in the graph, ideally all tasks should have this set correctly.